src/types/Todo.tsx

```diff
@@ -0,0 +1,12 @@
+import { User } from './User';
+
+export interface TodoWithoutUser {
+  id: number,
+  title: string,
+  completed: boolean,
+  userId: number,
+}
+
+export interface TodoWithUser extends TodoWithoutUser {
+  user: User | null;
+}
```

Comment:
```suggestion
export interface Todo {
  id: number,
  title: string,
  completed: boolean,
  userId: number,
}

export interface TodoWithUser extends Todo {
  user: User | null;
}
```
Maybe it will be more readable.